### ISSUE TITLE
[Question] Question on scaling: I am using GCN as a FE to train on a small observation space and test it on a larger observation space

### ISSUE THREAD

**Issue by hsaseendran** [DLR-RM/stable-baselines3#2042]
### ❓ Question

I am looking to scale my volt-var control model, which utilizes a Graph Convolutional Network (GCN) as a feature extractor. This approach is effective because the GCN adapts to changes in input dimensions, facilitating scalability.

Currently, I am training the model on a 13-bus system. After saving the model, I load and test it on a 34-bus system. The two systems have different shapes of observation and action spaces; the 13-bus system has smaller dimensions, while the 34-bus system has larger ones. For example, the observation space for the 13-bus system is (17, 3) for nodes and (2, 17) for edges, while the 34-bus system has an observation space of (47, 3) and edges of (2, 47).

The challenge I face is that the Stable Baselines3 (SB3) library requires me to predefine my observation space during initialization, which I would prefer to avoid. Because I don't know how big my observation space will become. My goal is to apply the learned model parameters from the 13 bus, such as weights, learning rate and other hyperparameters, to the 34-bus system

Has anyone worked on the scaling problem before? It doesn't have to be PowerGrid. Any example that demonstrates use of GCN with SB3 for scaling would be appreciated. Thank you! 

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo
- [X] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/)
- [X] If code there is, it is [minimal and working](https://github.com/DLR-RM/stable-baselines3/issues/982#issuecomment-1197044014)
- [X] If code there is, it is formatted using the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.
